src/ambianic/pipeline/store.py

```diff
@@ -171,18 +175,52 @@ def process_sample(self, **sample) -> Iterable[dict]:
                 yield processed_sample
 
     def notify(self, save_json: dict):
-        if self.notification is None:
-            return
-        log.debug("Sending notification(s)..")
         # TODO extract inference data
+        if save_json['inference_result'] is None:
+            return
+
         for inference_result in save_json['inference_result']:
             data = {
                 'id': save_json['id'],
                 'label': inference_result['label'],
                 'confidence': inference_result['confidence'],
                 'datetime': save_json['datetime'],
             }
-            notification = Notification(data=data, providers=self.notification_config["providers"])
+
+            try:
+                premiumFile = pkg_resources.resource_filename(
+                    "ambianic.webapp", "premium.yaml")
+                with open(premiumFile, "r") as file:
+                    configFile = yaml.safe_load(file)
+                    userId = configFile['credentials']["USER_AUTH0_ID"]
+                    endpoint = configFile['credentials']["NOTIFICATION_ENDPOINT"]
+                    if (userId):
+                        post(
+                            url='{0}/notification'.format(endpoint),
+                            json={
+                                'userId': userId,
+                                'notification': {
+                                    'title': 'Ambianic.ai New {0} event'.format(
+                                        data['label']),
+                                    'message': 'New {0} detected with a {1} confidence level'.format(
+                                        data['label'],
+                                        data['confidence']),
+                                }})
+
+            except FileNotFoundError:
+                log.info("Error locating file")
```

Comment:
@vickywane how would the user know if they missed a notification? Local logging is good for debugging, but how are we going to inform the user that edge is unable to send notifications?

Reply:
Logging of the sent notifications is an advanced feature which I would treat in a separate pull request. 

On the PWA side, I plan to refactor the settings page to contain sections about the notification and subscription feature. Logging from the edge device would be among the item it contains.

Reply:
@vickywane 
Is there an open issue for this future work on proper error handling when notifications fail to be sent?

Reply:
Yes, [this issue](https://github.com/ambianic/ambianic-subscriptions/issues/12) has been opened and its implementation would begin when this pull request is merged.